schemas/json/block.json

```diff
@@ -859,6 +859,17 @@
 		"render": {
 			"type": "string",
 			"description": "Template file loaded on the server when rendering a block."
+		},
+		"__experimentalAutoInsert": {
+			"type": "object",
+			"description": "Blocks to auto-insert this block next to.",
+			"patternProperties": {
+				"[a-zA-Z]": {
+					"type": "string",
+					"description": "Position relative to the block to auto-insert this block next to.",
+					"enum": [ "before", "after", "firstChild", "lastChild" ]
+				}
+			}
```

Comment:
Note to self: I'm not sure we really need the schema changes in order to expose the auto insert field; we might be able to remove them for the time being.

Reply:
Let's wait until the field becomes stable so we don't promote experimental features through autocompletion in IDE.

Reply:
Yeah, I only added this here since I thought it might be a requirement to get the field to be exposed by the block types REST API endpoint and corresponding client logic (see [our other discussion](https://github.com/WordPress/gutenberg/pull/52969#discussion_r1288224700)). Happy to remove it if it's not actually needed 😊

Reply:
Removed in https://github.com/WordPress/gutenberg/pull/52969/commits/5254c12150cf1803a67faf257759d40e7f8851a9.